DMX.Sdk/Brokers/DmxApis/DmxApiBroker.Labs.cs

```diff
@@ -0,0 +1,27 @@
+﻿// --------------------------------------------------------------- 
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// ---------------------------------------------------------------
+
+using DMX.Sdk.Models.Services.Foundations.Labs;
+
+namespace DMX.Sdk.Brokers.DmxApis
+{
+    public partial class DmxApiBroker
+    {
+        private const string LabsRelativeUrl = "api/labs";
+
+        public async ValueTask<List<Lab>> GetAllLabsAsync()
+        {
+            await GetAccessTokenForScope("GetAllLabs");
+
+            return await GetAsync<List<Lab>>(LabsRelativeUrl);
+        }
```

Comment:
Each one of these functions `GetAllLabsAsync` and `PostLabAsync` requires their own PRs